styles/layout.css

```diff
@@ -0,0 +1,26 @@
+#filters  {
+  list-style: none;
+  padding: 0;
+}
+
+#filters select {
+  width: 60%;
+  margin: .2em 2%;
+}
+
+@media only screen and (min-width: 640px) {
+  .site-header {
+    padding: .1em .1em .1em 1em;
+    float: none;
+  }
+
+  #filters select {
+    width: 15em;
+    margin: .2em .2em;
+  }
+
+  #filters li {
+    width: 20em;
+    display: inline;
+  }
+}
```

Comment:
Try and use classes when possible for CSS styling. ID styling doesn't scale well.